### DISCUSSION TITLE
Allow for use without https

### DISCUSSION THREAD

**Discussion by yllekz** [dani-garcia/vaultwarden#2388] [Ideas]
Hello, I attempted to deploy this to my private NAS before but found I was unable to register an account because the suite requires HTTPS to be implemented before doing so. I have no intention of ever exposing this to the internet or anywhere outside of my internal LAN and would like this limitation removed. Is this possible?

**Comment by redpixy** [comment]
you may want to look at this.
https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS

**Comment by RealOrangeOne** [comment]
This unfortunately isn't possible, due to restrictions the browser (righty) imposes on some of its features. See https://github.com/dani-garcia/vaultwarden/discussions/2274 for an older discussion.
